### PR TITLE
Fix numbering of interrupt lines for 8018X

### PR DESCRIPTION
**Pull request by mfld-fr**
See discussion in PR #84 about interrupt line numbering.

@cocus @robots : could you please review the changes ?

### CODE REVIEW

**Comment by robots** [comment]
It looks good.

**Comment by cocus** [comment]
I think these are fine! I was using an -EB as well, so no wonder why some stuff wasn't working on the emu86 but it worked on the phisical hardware. Thank you.

File: int-8018x.c [hunk 1]
```diff
@@ -79,7 +78,15 @@ int int_io_read (word_t p, word_t * w)
 		{
 			word_t out = 0;
 			for (size_t i = 0; i < 8; i++) {
-				if (_int_mask[i]) {
+				if (i == 0) {
```
[the remark this answers is not in the document]
Comment (mfld-fr) [reply int-8018x.c]: Added such defines in commit 65f8297.

**Comment by mfld-fr** [comment]
Thanks @robots and @cocus for the review !

**Pull request merged**